python/cli/prelude_cli/views/interactive.py

```diff
@@ -873,7 +873,9 @@ def __init__(self, wiz: Wizard, title: str):
         self.title = title
 
     def enter(self):
-        webbrowser.open('https://platform.preludesecurity.com/detect', new=2)
+        account = self.wiz.detect.account.headers['account']
+        token = self.wiz.detect.account.headers['token']
+        webbrowser.open(f'https://{self.wiz.detect.account.hq}/iam/user?account={account}&token={token}', new=2)
```

Comment:
will account.hq not already have the https:// in it?

Reply:
Correct; opening a PR
https://github.com/preludeorg/libraries/pull/519